src/Microsoft.VisualStudio.Services.Agent/AgentServer.cs

```diff
@@ -28,6 +31,9 @@ public interface IAgentServer : IAgentService
         // job request
         Task<TaskAgentJobRequest> RenewAgentRequestAsync(int poolId, long requestId, Guid lockToken, CancellationToken cancellationToken = default(CancellationToken));
         Task<TaskAgentJobRequest> FinishAgentRequestAsync(int poolId, long requestId, Guid lockToken, DateTime finishTime, TaskResult result, CancellationToken cancellationToken = default(CancellationToken));
+
+        // ReleaseManagement
+        IEnumerable<AgentArtifactDefinition> GetReleaseArtifactsFromService(Guid teamProject, int releaseId, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
+1 the server classes are just a thin wrapper over the http client to create an abstraction point for testability. A month or so back I remember discussing this a bit with at least a couple people... i forget who now... and we ended up concluding that it was better to have multiple server classes for readability. Otherwise we end up with one giant class with a bunch of unrelated methods (confusing and hard to pick the right one).